int32 pin
int32 stat
---
int32 out